src/styles/nav.scss

```diff
@@ -0,0 +1,38 @@
+.nav {
+  display: flex;
+
+  &__item {
+    display: inline-block;
+    list-style-type: none;
+  }
+
+  .nav__list {
```

Comment:
Why did you use the rule with nesting?
Now you have a selector: `.nav .nav__list`. Do you really want to create such a selector?
I think it's better to use here the next selector:
```
&__list {
```